build-tools/packages/build-cli/src/library/releaseTag.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * A TSDoc tag representing the release level of an API.
```

Comment:
Linking https://fluidframework.com/docs/build/releases-and-apitags#api-support-levels (or the source for that if its in the repo) which documents what these mean would be good.